source/common/runtime/runtime_features.cc

```diff
@@ -64,6 +64,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.allow_response_for_timeout",
     "envoy.reloadable_features.consume_all_retry_headers",
     "envoy.reloadable_features.check_ocsp_policy",
+    "envoy.reloadable_features.delta_ttl",
```

Comment:
Why a runtime flag?

Reply:
Holdover from when we gated the delta one on a flag due to perf concerns, I'll remove